plugins/aja/aja-source.cpp

```diff
@@ -293,6 +293,20 @@ void AJASource::CaptureThread(AJAThread *thread, void *data)
 				audioOverrun = true;
 			}
 
+			if (offsets.currentAddress - offsets.readOffset >
```

Comment:
Just to make it easier to understand the flow, perhaps move this check just in front of the `DMAReadAudio` call that it is checking against (Line 315 below in your code).

Reply:
Ok, I will update the code later.